src/style.css

```diff
@@ -0,0 +1,116 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  width: 60%;
+  margin: 5% 20%;
+  border: 1px solid #e2dbdb;
+}
+
+.cont {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #e2dbdb;
+  padding: 24px;
+}
+
+.refresh {
+  background: url('/src/assets/refresh.png');
+  background-size: contain;
+  opacity: 0.8;
+}
+
+button {
+  background: transparent;
+  border: none;
+  height: 24px;
+  width: 24px;
+  cursor: pointer;
+}
+
+button:active {
+  transform: scale(1.1);
+}
+
+.enter {
+  background: url('/src/assets/enter.png');
+  background-size: contain;
```

Comment:
- [ ] Good work so far, but please  avoid CSS rule repetition  instead define global rule or use CSS custom properties (--*) for good practice and easy maintenance.
 for example
 :root {
  --bg-size: contain;
}